Makefile

```
@@ -132,7 +132,11 @@ endif
 endif
 
 ## Ensures NPM dependencies are installed without having to run this all the time.
-webapp/node_modules: webapp/package.json
+ifneq ($(HAS_WEBAPP),)
+PACKAGE_JSON = webapp/package.json
+endif
+
+webapp/node_modules: $(PACKAGE_JSON)
```

Alternatively: 
```suggestion
webapp/node_modules: $(wildcard webapp/package.json)
```

From https://github.com/mattermost/mattermost-plugin-starter-template/blob/f5cae51a20a81dd87e11a793a3bb494d6800c9c0/Makefile#L79